Resources/Prototypes/Entities/Structures/Wallmounts/switch.yml

```diff
@@ -509,6 +509,333 @@
+  parent: BigRedButton
+  suffix: Brig
+  components:
+  - type: AccessReader
+    access: [["Brig"]]
+
+- type: entity
+  id: BigRedButtonLawyer
+  parent: BigRedButton
+  suffix: Lawyer
+  components:
+  - type: AccessReader
+    access: [["Lawyer"]]
+
+- type: entity
+  id: BigRedButtonEngineering
+  parent: BigRedButton
+  suffix: Engineering
+  components:
+  - type: AccessReader
+    access: [["Engineering"]]
+
+- type: entity
+  id: BigRedButtonMedical
+  parent: BigRedButton
+  suffix: Medical
+  components:
+  - type: AccessReader
+    access: [["Medical"]]
+
+- type: entity
+  id: BigRedButtonQuartermaster
+  parent: BigRedButton
+  suffix: Quartermaster
+  components:
+  - type: AccessReader
+    access: [["Quartermaster"]]
+
+- type: entity
+  id: BigRedButtonSalvage
+  parent: BigRedButton
+  suffix: Salvage
+  components:
+  - type: AccessReader
+    access: [["Salvage"]]
+
+- type: entity
+  id: BigRedButtonCargo
+  parent: BigRedButton
+  suffix: Cargo
+  components:
+  - type: AccessReader
+    access: [["Cargo"]]
+
+- type: entity
+  id: BigRedButtonResearch
+  parent: BigRedButton
+  suffix: Research
+  components:
+  - type: AccessReader
+    access: [["Research"]]
+
+- type: entity
+  id: BigRedButtonService
+  parent: BigRedButton
+  suffix: Service
+  components:
+  - type: AccessReader
+    access: [["Service"]]
+
+- type: entity
+  id: BigRedButtonMaintenance
+  parent: BigRedButton
+  suffix: Maintenance
+  components:
+  - type: AccessReader
+    access: [["Maintenance"]]
+
+- type: entity
+  id: BigRedButtonExternal
+  parent: BigRedButton
+  suffix: External
+  components:
+  - type: AccessReader
+    access: [["External"]]
+
+- type: entity
+  id: BigRedButtonJanitor
+  parent: BigRedButton
+  suffix: Janitor
+  components:
+  - type: AccessReader
+    access: [["Janitor"]]
+
+- type: entity
+  id: BigRedButtonTheatre
+  parent: BigRedButton
+  suffix: Theatre
+  components:
+  - type: AccessReader
+    access: [["Theatre"]]
+
+- type: entity
+  id: BigRedButtonBar
+  parent: BigRedButton
+  suffix: Bar
+  components:
+  - type: AccessReader
+    access: [["Bar"]]
+
+- type: entity
+  id: BigRedButtonChemistry
+  parent: BigRedButton
+  suffix: Chemistry
+  components:
+  - type: AccessReader
+    access: [["Chemistry"]]
+
+- type: entity
+  id: BigRedButtonKitchen
+  parent: BigRedButton
+  suffix: Kitchen
+  components:
+  - type: AccessReader
+    access: [["Kitchen"]]
+
+- type: entity
+  id: BigRedButtonChapel
+  parent: BigRedButton
+  suffix: Chapel
+  components:
+  - type: AccessReader
+    access: [["Chapel"]]
+
+- type: entity
+  id: BigRedButtonHydroponics
+  parent: BigRedButton
+  suffix: Hydroponics
+  components:
+  - type: AccessReader
+    access: [["Hydroponics"]]
+
+- type: entity
+  id: BigRedButtonAtmospherics
+  parent: BigRedButton
+  suffix: Atmospherics
+  components:
+  - type: AccessReader
+    access: [["Atmospherics"]]
+
+- type: entity
+  id: BigRedButtonEmergency
```

Comment:
just make it inherit signal button to remove copypaste, theres uses for also linking it to e.g. shutters

Reply:
wanted to do this, but inheriting signal button on the emergency one specifically overrides the alert system functionality :(

Reply:
if alert system one doesnt handle the trigger event it should be fine

Reply:
the issue was that they both set args to handled, so actually, if thats not a thing anymore it should work now. will look into it